agent/pkg/tunnel/protocol/tcp/tcpproxy.go

```diff
@@ -148,9 +148,9 @@ func (tp *TCPProxyService) GetProxyStream(targetNodeName, targetIP string, targe
 
 	msg.Reset()
 	if err = streamReader.ReadMsg(msg); err != nil {
-		err = stream.Reset()
-		if err != nil {
-			return nil, fmt.Errorf("Stream between %s reset err: %v", targetNodeName, err)
+		error := stream.Reset()
```

Comment:
error is a built-in golang type, use err1 replace it.

Reply:
done